src/hooks/users.ts

```diff
@@ -28,7 +110,7 @@ const useConnectVenueUsers = () => {
 export const useVenueUsers = (): readonly WithId<User>[] => {
   useConnectVenueUsers();
 
-  return useSelector(usersSelector) ?? [];
+  return useSelector(universeUsersSelector) ?? [];
 };
 
 export const usePartygoers = (): readonly WithId<User>[] => {
```

Comment:
With the `venueUsers.filter((user) => user.lastSeenAt > lastSeenThreshold)` filter below, I'm not sure if this is the correct logic.

I believe I called this out somewhere in https://github.com/sparkletown/sparkle/pull/1042, but I can't find it now/it must have gotten lost somewhere along the way.

Essentially, the old logic for this looked like:

```typescript
partygoers.filter(
        (partygoer) => partygoer?.lastSeenIn?.[venueName] > lastSeenThreshold
      ),
```

Why was the filter logic changed?

- old: `(partygoer) => partygoer?.lastSeenIn?.[venueName] > lastSeenThreshold`
- new: `(user) => user.lastSeenAt > lastSeenThreshold`

Reply:
```suggestion
export const usePartygoers = (venueId: string): readonly WithId<User>[] => {
```

Reply:
We don't have venue users used anywhere in this PR. `worldUsers` ensures that users fetched are part of the current world. So, filtering by lastSeenAt seems more relevant. Also, I think lastSeenIn is somewhat broken. By switching the locations, you are not updating this object, but rather rewriting. Looks like a bug, will jump on it in the next `counting PR`

Reply:
As mentioned in a number of my review comments in https://github.com/sparkletown/sparkle/pull/1047#pullrequestreview-573000412 we want to add `useRecentVenueUsers` in this PR.

`usePartygoers` used to use `useVenueUsers`, and there are many places in the app that relied on that logic, that has been lost since they were all converted to using `useRecentWorldUsers`. We need to fix that in this PR.